lib/instrumentation/index.js

```diff
@@ -135,7 +136,10 @@ Instrumentation.prototype.addEndedSpan = function (span) {
       }
 
       agent.logger.debug('sending span %o', { span: span.id, parent: span.parentId, trace: span.traceId, name: span.name, type: span.type })
-      if (agent._transport) agent._transport.sendSpan(payload)
+      if (agent._transport) {
+        agent._metrics.getOrCreateCounter('agent.spans.sent').inc()
```

Comment:
We decided to delay the introduction of `agent.*` metrics, so this can just be reverted